src/SuperAdmin/ForceVerifyUserForm.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { superAdminUpdate } from '../APIInterfaces/BackendAPI/Requests/UserRequests';
+import logger from '../Utilities/Logger';
+import { useGlobalSnackbarContext } from '../Contexts/GlobalSnackbar';
+
+export const ForceVerifyUserForm: React.FC<any> = () => {
+    const { register, handleSubmit } = useForm<{forceVerifyEmail: string}>();
+    const setAlert = useGlobalSnackbarContext();
+
+    const submit = async (data: {forceVerifyEmail: string}) => {
+        try {
+            await superAdminUpdate({email: data.forceVerifyEmail});
+            setAlert?.({severity: 'success', message: `Updated ${data.forceVerifyEmail}`});
+        } catch(e) {
+            logger.error(e);
+            setAlert?.({severity: 'error', message: 'Failed to force verify. Check logs.'});
+        }
+    };
+
+    return <form onSubmit={handleSubmit(submit)} >
```

Comment:
I'd prefer if this is two steps, we already have the get user route, it would be good if we could lookup the information before blindly updating